apps/web/ee/lib/helpscout/provider.tsx

```diff
@@ -0,0 +1,10 @@
+import { FC } from "react";
+import { LiveChatLoaderProvider } from "react-live-chat-loader";
+
+const Provider: FC = ({ children }) => (
+  <LiveChatLoaderProvider providerKey={process.env.NEXT_PUBLIC_HELPSCOUT_KEY!} provider="helpScout">
+    <>{children}</>
+  </LiveChatLoaderProvider>
+);
+
+export default Provider;
```

Comment:
NIT @zomars we may wanna merge this file with the other provider and make it more generic